.storybook/preview.js

```diff
@@ -1,4 +1,5 @@
 import '../stylesheets/all.scss';
+import '../stylesheets/header.scss';
```

Comment:
What about having the header's style in the component folder and have it loaded in the components itself? 🤔